Bot/cogs/remindme.py

```diff
@@ -70,10 +70,13 @@ def cog_unload(self):
         if time_set < time_now:
             delta_time += timedelta(days=1)
             
-        await self.remindme(self,ctx,str(timedelta(seconds=int(delta_time.total_seconds()))),*,message="")
+        await self.remindme_base(self,ctx,str(timedelta(seconds=int(delta_time.total_seconds()))),*,message="")
         
     @commands.command(hidden=True,pass_context=True)
     async def remindme(self,ctx,get_time,*,message=""):
+        await self.remindme_base(self,ctx,get_time,*,message="")
```

Comment:
Syntax error.

That not how you call object function, you don't include `self` and no need to `,*,` just message = message.

I suggest you should check how python calling function work.

Also for Line 73, why bother doing Timedelta when delta_time is already Timedelta?